### DISCUSSION TITLE
Нормативная мощность для ФРМО.Organisation: CarePlan vs PlanDefinition

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#69] [Административная РГ]
ФРМО требует для организации указания нормативной мощности: для поликлиники - в посещениях или в прикрепленном контингенте, для стационара - в койках по определенным профилям.

Есть гипотеза, что нормативная мощность представляет собой намерение организации обслужить клиентов (контингента пациентов), оказав им определенное количество услуг в смену.
Таким образом, это план обслуживания контингента пациентов, такой, что:
- CarePlan.Subject = Group
- CarePlan.Intent = Plan
- CarePlan.Contributor = наша организация, если нормируется деятельность оргструктуры
- CarePlan.Activity.Detail.Location = наше здание, если нормируется объем деятельности в здании
- CarePlan.Activity.Reference = ServiceRequest, такой что
- ServiceRequest.Quantity = указанная мощность
- ServiceRequest.Code нужно профилировать на коечные профили

Была высказана (А.Иванов, О.Пензин) другая гипотеза: CarePlan обычно используется для конкретных пациентов, а для обсуждаемой абстрактной вещи может быть использован PlanDefinition.
Но при попытке его применить мне показалось, что он - не план, а правила создания планов.
Выношу на дискуссию.

**Comment by ai-fhir** [comment]
Да, действительно, PlanDefinition это описательный паттерн и он говорит, как породить CarePlan, но в момент обсуждения я неверно понял сам вопрос:

Я бы кодировал мощность как частный случай расписания - https://www.hl7.org/fhir/schedule.html и https://www.hl7.org/fhir/slot.html#Slot

Это довольно гибкий механизм, может быть слот с возможностью множественного букирования - когда у нас просто есть n талонов на промежуток времени. 

Само расписание можно вешать на Location, который может быть атомарным, вплоть до отдельной койки.

**Comment by eugenekogan-spb** [comment]
Мне кажется, на плановую мощность записаться нельзя. Это расчетный показатель, для бюджетирования и  сопоставления с реальным. 

С уважением, Евгений Коган 

27 дек. 2021 г., в 19:09, ai-fhir ***@***.***> написал(а):

﻿
Да, действительно, PlanDefinition это описательный паттерн и он говорит, как породить CarePlan, но в момент обсуждения я неверно понял сам вопрос:

Я бы кодировал мощность как частный случай расписания - https://www.hl7.org/fhir/schedule.html и https://www.hl7.org/fhir/slot.html#Slot

Это довольно гибкий механизм, может быть слот с возможностью множественного букирования - когда у нас просто есть n талонов на промежуток времени.

Само расписание можно вешать на Location, который может быть атомарным, вплоть до отдельной койки.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by ai-fhir** [reply]
Я предполагал, что плановая мощность - число слотов, реальное - число забронированых слотов.

**Comment by eugenekogan-spb** [comment]
Думаю, нет. Вы строите поликлинику на 1000 посещений в день. Из этого считаете штатное расписание, оборудование и т.д. Потом в реальное расписание попадает на этот месяц столько-то талонов к терапевту, столько к хирургу. На другой месяц иначе. 

С уважением, Евгений Коган 

27 дек. 2021 г., в 19:28, ai-fhir ***@***.***> написал(а):

﻿
Я предполагал, что плановая мощность - число слотов, реальное - число забронированых слотов.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
